### ISSUE TITLE
Getting "Error: EEXIST: file already exists" on npm start

### ISSUE THREAD

**Issue by bartonwelt** [bengsfort/rollup-plugin-copy-assets#1]
Hello. I came across this plugin as I was looking for a solution to copy font files from my src folder to dist on build. Originally, the plugin was working great, but on consecutive npm start attempts I'm running into the following error:

```[!] Error: EEXIST: file already exists, mkdir '/Users/workspace/front-end/project/dist/fonts'```

I added a prestart script to remove the dist folder, but I still get the error. Do you have any idea what the issue could be?

My plugin config looks like this:
```
plugins: [
  copy({
    assets: ['src/fonts']
  })
]
```

```
node v8.11.1
npm v5.8.0
```

**Comment by mrjasonroy** [comment]
Getting the same here

`npm v6.1.0
 node v6.11.0`

**Comment by bengsfort** [comment]
Will look into this later today.

**Comment by atherdon** [comment]
I have a similar issue. Btw, thanks for this plugin!
`npm 6.7.0 node v.11.6.0`

**Comment by atherdon** [comment]
@bengsfort sorry for interrupting you, I saw that you made some change in Feb - does it cover this issue?

**Comment by musicbender** [comment]
Same issue here. Do you all have multiple objects in your rollup output? This problem was fixed when I only included one output:

`output: [
    {
      external,
      file,
      format: 'umd',
      globals,
      name
    },
    {
      external,
      file: fileMod,
      format: 'es',
      globals,
      name
    }
  ],`

changed to:
`output: [
    {
      external,
      file,
      format: 'umd',
      globals,
      name
    }
  ],`

Perhaps it works for the first build, but when it goes through the second the dir is already there from the first build?

**Comment by bengsfort** [comment]
tldr; this should be fixed in 2.0.0. If it is still occurring, please let me know and provide a rollup config so I can investigate further.

I was only able to recreate this once, and when it happened the changes made had fixed it for my instance. It is now set up to check whether the file/directories exist and if so remove them before continuing. I have added tests that pre-emptively add all of the assets and directories to the test output directory, then run a rollup build and that is working perfectly. If there is more concrete info on how to reproduce this I will gladly re-open the ticket and investigate more, but as of now I cannot reproduce.
